client/components/payment-delete-illustration/index.tsx

```diff
@@ -10,7 +11,10 @@ import classNames from 'classnames';
  */
 import './styles.scss';
 
-const PaymentDeleteIllustration = ( { hasBorder, Icon = () => null } ) => {
+const PaymentDeleteIllustration: React.FunctionComponent< {
```

Comment:
It would be great to introduce a test for this component (a snapshot will work as we have nothing yet).

Reply:
Good idea! I added tests in https://github.com/Automattic/woocommerce-payments/pull/3910/commits/473fc2521ceedd21ac3f9a731e157f578b846e93